Packer: Update AMI server-setup.json to avoid lockout

ukanga opened the pull request:
The apt-daily* services may sometimes prevent or slow down the
process of building an AMI image.

ukanga committed:
Packer: Update AMI server-setup.json to avoid lockout

The apt-daily* services may sometimes prevent or slow down the
process of building an AMI image.

jasonrogena commented on ansible.cfg:
```diff
@@ -2,6 +2,16 @@
+# Use Mitogen connection backend by default for faster execution
+# See: https://mitogen.networkgenomics.com/ansible_detailed.html
+#strategy_plugins = .venv/lib/python3.7/site-packages/ansible_mitogen/plugins/strategy
+strategy = mitogen_linear
```
@ukanga we might need to add the Mitogen dependency in the https://github.com/OpenSRP/playbooks/blob/master/requirements/base.{in,pip} files.

ukanga replied:
Thank you @jasonrogena

jasonrogena committed:
Add Mitogen to pip requirements

Add the Mitogen pip package requirement to the base.pip and base.in
requirement files.

Signed-off-by: Jason Rogena <jason@rogena.me>

jasonrogena committed:
Set default strategy_plugins value, Mitogen docs

Add documentation for why we use Mitogen and how to override the default
value for the strategy_plugins, which has also been set in this commit.

Signed-off-by: Jason Rogena <jason@rogena.me>
